phpstan-for-fixtures.neon

```diff
@@ -0,0 +1,37 @@
+rules:
+    - Symplify\PHPStanRules\Rules\CheckClassNamespaceFollowPsr4Rule
+
+services:
+    # makes $container->make() generic type
+    - Rector\Utils\PHPStan\ClassConstFetchReturnTypeResolver
+    -
+        class: Rector\Utils\PHPStan\TypeExtension\ContainerMakeReturnTypeExtension
+        tags:
+            - phpstan.broker.dynamicMethodReturnTypeExtension
+
+parameters:
+    reportUnmatchedIgnoredErrors: false
+    customRulesetUsed: true
+
+    fileExtensions:
+        - inc
+
+    paths:
+        - packages-tests/
+        - rules-tests/
+
+    strictRules:
+        allRules: false
+
+    excludePaths:
+        - '*/Fixture/*'
+
+    ignoreErrors:
+        - '#Cognitive complexity .*#'
+        - '#Out of .* possible .* types .*#'
+        - '#.* is never used#'
+        - '#.* is forbidden to use#'
+        - '#Instead of .*#'
+        - '#.* cannot be used/left in the code: .*#'
+        - '#.* to respect parent type#'
+        - '#.* annotation with test case class reference#'
```

Comment:
Ideally, here should only run to check the namespace and class exists

Reply:
These rules auto register via phpstan extension installer.

I don't know a way how I can disable them just for this config

Reply:
I see. Not sure this check is worth it, as there are dozens of false positive and no improvement in the fixtures themselves.

Reply:
lets see, [whether ondrej accepts a new config switch to disable the extension installer](https://github.com/phpstan/phpstan/discussions/9902)

Reply:
That would be perfect :clap:

Reply:
instead of disabling rules, inspired by ondreys comment I remove the rule packages within the github action, so I don't need to ignore the errors at phpstan config